RallyHereIntegration/Source/RallyHereIntegration/Public/RH_EntitlementSubsystem.h

```diff
@@ -20,6 +20,12 @@ class URH_LocalPlayerSubsystem;
 DECLARE_DELEGATE_TwoParams(FRH_ProcessEntitlementCompletedDelegate, bool /*Success*/, FRHAPI_PlatformEntitlementProcessResult) 
 DECLARE_DELEGATE_RetVal(ERHAPI_PlatformRegion, FRH_GetPlatformRegionDelegate)
 
+UDELEGATE()
+DECLARE_DYNAMIC_DELEGATE_OneParam(FRH_QueryStoreOffersDynamicDelegate, bool, bSuccess);
```

Comment:
Unless you are planning to add more params here, there is a FRH_GenericSuccessBlock that has this signature for simple pass/fail operation delegates.